README.md

```diff
@@ -1,22 +1,24 @@
 # Akka Serialization Helper
 
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.virtuslab/sbt-akka-serialization-helper/badge.svg)](https://maven-badges.herokuapp.com/maven-central/cz.jirutka.rsql/rsql-parser)
```

Comment:
Reminder for me: to update after #104 is merged

Reply:
`cz.jirutka.rsql`? :trollface:

Reply:
Oh, sth with 0.3.1 version, I see xD